transactions/transactions_service.go

```diff
@@ -49,20 +49,44 @@ const (
 type Transaction = db.Transaction
 
 type Boostagram struct {
-	AppName        string `json:"app_name"`
-	Name           string `json:"name"`
-	Podcast        string `json:"podcast"`
-	URL            string `json:"url"`
-	Episode        string `json:"episode,omitempty"`
-	FeedId         string `json:"feedID,omitempty"`
-	ItemId         string `json:"itemID,omitempty"`
-	Timestamp      int64  `json:"ts,omitempty"`
-	Message        string `json:"message,omitempty"`
-	SenderId       string `json:"sender_id"`
-	SenderName     string `json:"sender_name"`
-	Time           string `json:"time"`
-	Action         string `json:"action"`
-	ValueMsatTotal int64  `json:"value_msat_total"`
+	AppName        string         `json:"app_name"`
+	Name           string         `json:"name"`
+	Podcast        string         `json:"podcast"`
+	URL            string         `json:"url"`
+	Episode        StringOrNumber `json:"episode,omitempty"`
+	FeedId         StringOrNumber `json:"feedID,omitempty"`
+	ItemId         StringOrNumber `json:"itemID,omitempty"`
+	Timestamp      int64          `json:"ts,omitempty"`
+	Message        string         `json:"message,omitempty"`
+	SenderId       StringOrNumber `json:"sender_id"`
+	SenderName     string         `json:"sender_name"`
+	Time           string         `json:"time"`
+	Action         string         `json:"action"`
+	ValueMsatTotal int64          `json:"value_msat_total"`
+}
+
+type StringOrNumber struct {
```

Comment:
Did you look at any other ways to do it, like Did you look at any other ways to do it? like https://stackoverflow.com/a/77624153/4562693 ? it seems `json.Number` might be a nice type so we don't need to implement it ourselves

Reply:
That's nice! But these fields are unpredictable right? What if it's a uuid?

Reply:
Ah, good point! it won't work for a uuid I guess.